web/styles/rendered_markdown.css

```diff
@@ -575,26 +597,6 @@
         margin-right: 5px;
     }
 
-    .twitter-image .media-image-element,
-    .message_inline_image .media-image-element,
-    .message_inline_ref .media-image-element {
-        /* We use `scale-down` so that images smaller than the container are
-           neither scaled up nor cropped to fit. This preserves
-           their aspect ratio, which is often helpful. */
-        object-fit: scale-down;
-
-        /* We need to explicitly specify the image dimensions to have
-           object-fit work; likely because internally object-fit needs
-           to know the frame it is targeting, and images don't default
-           to container dimensions. */
-        height: 100%;
-        width: 100%;
-
-        float: left;
-        margin-right: 10px;
-        border-radius: inherit;
-    }
```

Comment:
I guess these subtypes of images don't have thumbnails (they're just small in the first place). Have you tested how those get sized? (website previews are the easiest to generate).

Reply:
I've tested with externally linked images, which behave nicely, but let me check link previews.

Reply:
Website preview images are unaffected; those are handled care of the `.message_embed_image` selector (which needs a scaling refresh, at the very least, as it's still using 70px square images, but that should be a separate PR).